app/views/pieces/show.html.erb

```diff
@@ -0,0 +1,52 @@
+<h1>Pieces show page goes here</h1>
+
+<!-- <script>
+
+$(function() {
+  $('.square a').click(
+    function submitPieceMove(id, file, rank) {
+      // console.log(id);         // Error, the id is not passed, results in 
+                                  // wrong routing /pieces/[object%20Object]
+      $.ajax("/pieces/" + id, {
+        method: "PATCH",
+        data: {
+          file: file,
+          rank: rank
+        }
+      });
+  });
+});
+
+</script> -->
+
+<div class="chessboard">
+
+  <div class="pieces">
+    <% file = 0 %>
+    <% rank = 9 %>
+
+    <% 8.times do %>
+      <% rank = rank - 1 %>
+      <% 8.times do %>
+        <% file = file + 1 %>
+          <div class="square" id="<%= [file, rank] %>">
+            <% if @piece_coord == [file, rank] %>
+              <div class="highlight">
+                <p><%= "#{file}, #{rank}" %></p>
+              </div>
+            <% else %>
+              <div>
+                <!-- <a href='#' onclick="submitPieceMove(@current_piece.id, file, rank)"><%= "#{file}, #{rank}" %></a> -->
+                <%= button_to piece_path(@current_piece), method: :patch, params: {piece: {file: file, rank: rank}} do %>

```

Comment:
There seems to be strange glitch on the front-end where file 1 jumps to file 8.  Haven't had a chance to examine this carefully.

![button display glitch](https://user-images.githubusercontent.com/20831001/32452928-c86a4c8c-c2cf-11e7-842c-e9b6eaf12852.png)